### DISCUSSION TITLE
please help me

### DISCUSSION THREAD

**Discussion by LEESUNJAE1114** [community/community#63397] [Programming Help]

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @LEESUNJAE1114 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by elderguardian** [comment]
Check your pull requests on the repo

**Reply by LEESUNJAE1114** [reply]
너는 코딩으로 사이트를 만들어 본적이있니? 검색활동이 되는 사이트 같은 거. 혹시 나와 동업 할 생각이 있다면 답장줘!

**Comment by LEESUNJAE1114** [comment]
are you luca?
first. where are you from?
i am korean.
There is a way to make a website and sell it, so I've been trying to make
it myself, but it's not easy, so I'm looking for a developer. If you just
make a website, there is a way I can definitely attract customers.

2023년 8월 12일 (토) 오전 12:54, Crazy Bunny ***@***.***>님이 작성:
